103-bundle-runtime.md

```diff
@@ -195,13 +195,13 @@ For this example CNAB bundle:
   "images": {
     "my-microservice": {
       "description": "my microservice",
-      "digest": "sha256:aaaaaaaaaaaa...",
+      "content-digest": "sha256:aaaaaaaaaaaa...",
       "image": "technosophos/microservice:1.2.3"
     }
   },
   "invocationImages": [
     {
-      "digest": "sha256:aaaaaaa...",
+      "content-digest": "sha256:aaaaaaa...",
```

Comment:
I think there is one more reference of `digest` in this file, in the example below (line 240): "The `/cnab/app/image-map.json` file mounted in the invocation image will be"

Reply:
good catch. updated.